plugins/modules/remote_management/redfish/ilo_redfish_command.py

```diff
@@ -0,0 +1,341 @@
+    type: str
+  key_file:
+    required: false
+    description:
+      - absolute path to the server key file
+    type: str
+author:
+  - Gayathiri Devi Ramasamy (@Gayathirideviramasamy)
+  - T S Kushal (@TSKushal)
+  - Varni H P (@varini-hp)
+  - Prativa Nayak (@prativa-n)
+'''
+
+EXAMPLES = '''
+  - name: Verify service BIOS attributes
+    ilo_redfish_command:
+      category: Systems
+      command: VerifyServiceBiosAttributes
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      service_attributes:
+        ConnectSequenceTrace: "Disabled"
+        CustomDebugBlkIo: "Disabled"
+
+  - name: Verify bios attributes
+    ilo_redfish_command:
+      category: Systems
+      command: VerifyBiosAttributes
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "*****"
+      bios_attributes:
+        SubNumaClustering: "Disabled"
+        WorkloadProfile: "Virtualization-MaxPerformance"
+
+  - name: Verify logical drives
+    ilo_redfish_command:
+      category: Systems
+      command: VerifyLogicalDrives
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      raid_details: [{"LogicalDriveName": "LD1",
+                       "Raid": "Raid1",
+                       "DataDrives": {
+                          "DataDriveCount": 2,
+                          "DataDriveMediaType": "HDD",
+                          "DataDriveInterfaceType": "SAS",
+                          "DataDriveMinimumSizeGiB": 0
+                          }
+                      }]
+
+  - name: Verify UEFI boot order
+    ilo_redfish_command:
+      category: Systems
+      command: VerifyUefiBootOrder
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      uefi_boot_order: ["Generic.USB.1.1"]
+
+  - name: Verify specified logical drives
+    ilo_redfish_command:
+      category: Systems
+      command: VerifySpecifiedLogicalDrives
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      raid_details: [{"LogicalDriveName": "LD1",
+                       "Raid": "Raid1",
+                       "DataDrives": {
+                          "DataDriveCount": 2,
+                          "DataDriveMediaType": "HDD",
+                          "DataDriveInterfaceType": "SAS",
+                          "DataDriveMinimumSizeGiB": 0
+                          }
+                      }]
+
+  - name: Check server reboot status
+    ilo_redfish_command:
+      category: Systems
+      command: CheckRebootStatus
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+
+  - name: Get specified logical drives details
+    ilo_redfish_command:
+      category: Systems
+      command: CheckRebootStatus
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      logical_drives_names: ["LD1", "LD2"]
+
+'''
+
+RETURN = '''
+ilo_redfish_command:
```

Comment:
why return one single dictionary with the data inside instead of directly returning the data?

Reply:
Can you elaborate on this comment?